### ISSUE TITLE
Unable to deploy to Cloud Foundry

### ISSUE THREAD

**Issue by BenoitVerdier** [travis-ci/dpl#408]
I'm unable to deploy to Cloud Foundry from Travis since yesterday, even though the 'cf push' is working as intended on my computer.
I ran the 'dpl' command on my computer, but I never get any error (even with an invalid email, password or organization)
```
dpl --provider=cloudfoundry --username=EMAIL --password=PASSWORD --organization=ORG --api=api.run.pivotal.io --space=development --skip-ssl-validation
Installing deploy dependencies
Preparing deploy
Deploying application
```
I also tried to restart a previous build from Travis UI (one that had been deploy with success), but still no deploy.

**Comment by Michaeldean147** [comment]
We are also having this same issue.

**Comment by BanzaiMan** [comment]
Do you have a build log URL to show this problem?

**Comment by Michaeldean147** [comment]
The command "npm test" exited with 0.
    Fetching: dpl-1.8.13.travis.1306.4.gem (100%)

    Installing deploy dependencies

    Preparing deploy

    Deploying application

    Done. Your build exited with 0.`

After this is completed nothing is updated on the CF side.

**Comment by BenoitVerdier** [comment]
here is the log with 'edge: true' https://travis-ci.org/3IE/TypescriptAngularStarter/builds/111172190
and the log with the 'edge: false' https://travis-ci.org/3IE/TypescriptAngularStarter/builds/111154812

**Comment by xtreme-gabelan-satchithanantham** [comment]
I'm seeing the same thing with our builds. It stopped deploying after our first Pull Request (We were merging manually and pushing before)

**Comment by BanzaiMan** [comment]
Looks like Cloud Foundry Go tools URL has changed.

```sh-session
$ wget http://go-cli.s3-website-us-east-1.amazonaws.com/releases/latest/cf-linux-amd64.tgz 
--2016-02-24 14:55:42--  http://go-cli.s3-website-us-east-1.amazonaws.com/releases/latest/cf-linux-amd64.tgz
Resolving go-cli.s3-website-us-east-1.amazonaws.com... 54.231.14.4
Connecting to go-cli.s3-website-us-east-1.amazonaws.com|54.231.14.4|:80... connected.
HTTP request sent, awaiting response... 404 Not Found
2016-02-24 14:55:45 ERROR 404: Not Found.
```

**Comment by BanzaiMan** [comment]
We should be indicating errors like this more prominently (but that's a different issue).

**Comment by BanzaiMan** [comment]
OK. I've changed the URL to https://cli.run.pivotal.io/stable?release=debian64&source=github on the edge. Could you try it with:

```yaml
deploy:
  provider: cloud_foundry
  edge: true
  ...
```

Thanks. If it's successful, I'd push a new release.

**Comment by BenoitVerdier** [comment]
It did have an impact, but it's not working yet
```
Fetching: dpl-1.8.13.travis.1311.4.gem (100%)
Successfully installed dpl-1.8.13.travis.1311.4
1 gem installed

Installing deploy dependencies

Preparing deploy

gzip: stdin: not in gzip format
tar: Child returned status 1
tar: Error is not recoverable: exiting now

Deploying application

Done. Your build exited with 0.
```

**Comment by BanzaiMan** [comment]
Ooops. That was a URL for the `.deb` archive, not the tarball. I've fixed that on the edge. Could you try again? Thanks.

**Comment by BenoitVerdier** [comment]
Yeah, it's working for me :)
Thx

**Comment by BanzaiMan** [comment]
Released 1.8.13 with this fix. Thanks!
